suggestedimages/routes.py

```diff
@@ -2,36 +2,50 @@
     Blueprint, flash, redirect, render_template, request, url_for
 )
 import urllib
+from collections import namedtuple
+
 
 from . import search
 from .util import StrInLanguage
 from .locales import Locale
 
+LangOption = namedtuple("LangOption", "value label")
+
 bp = Blueprint('main', __name__)
 
 @bp.route('/', methods=('GET',))
 def index():
```

Comment:
This route is a bit bloated but it will be an interesting task to make it more readable. Don't really see any unnecessary code here.

Reply:
I made it less bloated by extracting code into functions.